app/views/admin/content_assets/index.html.erb

```diff
@@ -1,25 +1,36 @@
 <h1  class="govuk-heading-l">Assets</h1>
+<%= govuk_button_link_to 'Add asset', new_admin_content_asset_path, class: "govuk-button" %>
 
-<%= link_to 'Add asset', new_admin_content_asset_path, :class => "govuk-button" %>
+<%= govuk_accordion do |accordion|
+  get_all_content_pages.each do |content_page|
+    accordion.section(heading_text: content_page.title) do %>
 
-<table class="govuk-table">
-  <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Title</th>
-      <th scope="col" class="govuk-table__header">Alt Text</th>
-      <th scope="col" class="govuk-table__header"></th>
-    </tr>
-  </thead>
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">Image</th>
+            <th scope="col" class="govuk-table__header govuk-!-width-one-half">Name</th>
+            <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">Action</th>
+          </tr>
+        </thead>
 
-  <tbody class="govuk-table__body">
-    <% @content_assets.each do |content_asset| %>
-      <tr class="govuk-table__row">
-        <td class="govuk-table__cell" ><%= content_asset.title %></td>
-        <td class="govuk-table__cell" ><%= content_asset.alt_text %></td>
-        <td class="govuk-table__cell"><%= link_to 'Edit', edit_admin_content_asset_path(content_asset), { :class => "govuk-link" } %></td>
-      </tr>
+        <tbody class="govuk-table__body">
+          <% @content_assets.each do |content_asset| %>
+            <% if content_asset.page_name.to_i == content_page.position %>
+              <tr class="govuk-table__row">
+                <td class="govuk-table__cell"><%= image_tag(url_for(content_asset.asset_file), class: 'govuk-!-width-full') %></td>
+                <td class="govuk-table__cell" ><%= content_asset.title %></td>
+                <td class="govuk-table__cell">
+                  <%= govuk_button_link_to 'Edit', edit_admin_content_asset_path(content_asset), secondary: true %>
+                  <%= button_tag "Copy to clipboard", type: 'button', class: "govuk-button govuk-button--secondary", onClick: "navigator.clipboard.writeText('#{polymorphic_url(content_asset.asset_file)}')" %>
```

Comment:
Pedantic, but lowercase on click key works fine and matches rails contentions (to use snake case rather than camel). Also I'd suggest splitting this over a few lines to make it clearer:
```ruby
 <%= button_tag(
       "Copy to clipboard", 
       type: 'button', 
       class: "govuk-button govuk-button--secondary", 
       onclick: "navigator.clipboard.writeText('#{polymorphic_url(content_asset.asset_file)}')"
     ) %>
```

Reply:
i've made this change